ida/data.yml

```diff
@@ -7290,17 +7308,87 @@ classes:
     vtbls:
       - ea: 0x141A51090
         base: Component::GUI::AtkUnitBase
+    funcs:
+      0x140EA9DE0: ctor
   Client::UI::AddonCharacterStatus:
     vtbls:
       - ea: 0x141A512C8
         base: Component::GUI::AtkUnitBase
+    funcs:
+      0x140EAD350: ctor
+  Client::UI::AddonCharacterProfile:
+    vtbls:
+      - ea: 0x141A62250
+        base: Component::GUI::AtkUnitBase
+    funcs:
+      0x140F5BA20: ctor
+  Client::UI::AddonCharacterClass:
+    vtbls:
+      - ea: 0x141A62488
+        base: Component::GUI::AtkUnitBase
+    funcs:
+      0x140F5C310: ctor
+  Client::UI::AddonCharacterRepute:
+    vtbls:
+      - ea: 0x141A626C0
+        base: Component::GUI::AtkUnitBase
+    funcs:
+      0x140F5D3D0: ctor
+  Client::UI::AddonRecommendEquip:
+    vtbls:
+      - ea: 0x141A6F778
+        base: Component::GUI::AtkUnitBase
+    funcs:
+      0x140FCD860: ctor
+  Client::UI::AddonAdventureNoteBook:
+    vtbls:
+      - ea: 0x141A54270
+        base: Component::GUI::AtkUnitBase
+    funcs:
+      0x140ED52C0: ctor
+  Client::UI::AddonAetherCurrent:
+    vtbls:
+      - ea: 0x141A6F308
+        base: Component::GUI::AtkUnitBase
+    funcs:
+      0x140FCB060: ctor
+  Client::UI::AddonFateProgress:
+    vtbls:
+      - ea: 0x141A694A0
+        base: Component::GUI::AtkUnitBase
+    funcs:
+      0x140F99710: ctor
+  Client::UI::AddonFishingNote:
+    vtbls:
+      - ea: 0x141A534E8
+        base: Component::GUI::AtkUnitBase
+    funcs:
+      0x140ECB230: ctor
+  Client::UI::AddonFishRecord:
+    vtbls:
+      - ea: 0x141A53BA8
+        base: Component::GUI::AtkUnitBase
+    funcs:
+      0x140ED1410: ctor
+  Client::UI::AddonFishGuide2:
+    vtbls:
+      - ea: 0x141A53738
+        base: Component::GUI::AtkUnitBase
+    funcs:
+      0x140ECE610: ctor
+  Client::UI::AddonGearSetList:
+    vtbls:
+      - ea: 0x141A65F40
+        base: Component::GUI::AtkUnitBase
+    funcs:
+      0x140F76F00: ctor
   Client::UI::AddonMonsterNote:
     vtbls:
       - ea: 0x141A526E0
         base: Component::GUI::AtkUnitBase
     funcs:
       0x140EB5810: ctor
-  Client::UI::AddonGatheringNoteBook:
+  Client::UI::AddonGatheringNote:
```

Comment:
this is `AddonGatheringNoteBook`
addons are usually named after what is used in `LoadUldResourceHandle` or the name from the old RTTI

Reply:
I renamed this because `/tweaks debug` said it's called `AddonGatheringNote`. 🤷‍♂️

Reply:
Hmm yeah, classinformer.csv has it as `AddonGatheringNoteBook`. I'll change it back.